diesel_cli/src/cli.rs

```diff
@@ -145,6 +147,29 @@ pub fn build_cli() -> Command {
                         .default_missing_value("NOT_SET")
                         .num_args(0..=1)
                         .require_equals(true),
+                )
+                .arg(
+                    Arg::new("table-name")
+                        .index(2)
+                        .num_args(1..)
+                        .action(clap::ArgAction::Append)
+                        .help("Table names to filter (default only-tables if not empty)."),
```

Comment:
@weiznich would you like me to change this to:
```suggestion
                        .help("Table names to filter"),
```

Because as far as I can see, filtering so far has only worked by specifying `-o` or `-e`
the change would also be made for print-schema [below](https://github.com/diesel-rs/diesel/pull/3725/files#diff-324fe63179d6dfa0bc37bfbf038adadd1782c3448c9034c1142fef59b8984bf7R226)

Reply:
I'm in favor of this change (also for the print-schema) part.

Reply:
No worries from taking some time. We all work on this in our free time, so it's fine to take as much time as needed. There is always real life, which is more important than this.